plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py

```diff
@@ -1,5 +1,9 @@
 from typing import TYPE_CHECKING, List, Optional, Union
 
+from pygments import highlight
+from pygments.formatters.html import HtmlFormatter
+from pygments.lexers.python import PythonLexer
```

Comment:
could we move these to line 41? so it won't always import pygments when people use other renderer in this file.

Reply:
updated